File: CS/GridDemo.Wpf/VehiclesData.cs
using DevExpress.DemoData.Models.Vehicles;
using System.Collections.Generic;
using System.Linq;
using VehicleModel = DevExpress.DemoData.Models.Vehicles.Model;

namespace DevExpress.DemoData {
    public class VehiclesData {
        VehiclesContext context = new VehiclesContext();
        public IEnumerable<VehicleModel> Models {
            get { return context.Models.ToList(); }
        }
    }
}

File: VB/GridDemo.Wpf/VehiclesData.vb
Imports DevExpress.DemoData.Models.Vehicles
Imports System.Collections.Generic
Imports System.Linq
Imports VehicleModel = DevExpress.DemoData.Models.Vehicles.Model

Namespace DevExpress.DemoData
    Public Class VehiclesData
        Private context As New VehiclesContext()
        Public ReadOnly Property Models() As IEnumerable(Of VehicleModel)
            Get
                Return context.Models.ToList()
            End Get
        End Property
    End Class
End Namespace
